lib/oli_web/controllers/session_controller.ex

```diff
@@ -0,0 +1,23 @@
+defmodule OliWeb.SessionController do
+  use OliWeb, :controller
+
+  import Pow.Phoenix.Controller, only: [require_authenticated: 2]
+
+  plug :require_authenticated when action in [:signout]
+
+  @session_data_to_delete [:dismissed_messages]
+
+  def signout(conn, %{"type" => type}) do
```

Comment:
I would like to move more methods from `DeliveryController` to this controller, but left it for tech debt so we avoid getting this PR bigger and bigger.

Reply:
💯